libs/model/src/main/java/de/cyface/model/MetaData.java

```diff
@@ -78,10 +84,12 @@ public class MetaData implements Serializable {
      * @param appVersion The version of the app that transmitted the measurement, such as "1.2.0" or "1.2.0-beta1".
      * @param length The length of the measurement in meters.
      * @param userId The id of the user who has uploaded this measurement.
-     * @param version The version of this {@code Measurement} model, such as "1.1.1" or "2.0.0".
+     * @param version The format version in which the {@code Measurement} was deserialized, e.g. "2.0.0".
      */
     public MetaData(final MeasurementIdentifier identifier, final String deviceType, final String osVersion,
             final String appVersion, final double length, final String userId, final String version) {
+
+        Validate.isTrue(version.matches(SUPPORTED_VERSIONS), "Unsupported version: %s", version);
```

Comment:
Had a bug in the code where version "1.0.0" was written when deserializing data, instead of the current version (2.0.0) and we did not catch this.